### PR TITLE
feat(api): Screenshots APIs: add new parameters support

### LINKED ISSUE #139: Screenshots APIs: add the `labelIds` parameter support
Several Screenshots API endpoints have been enhanced with the new parameter support: `labelIds`. It should be added to this API client.

**References:**

- [Add Screenshot](https://developer.crowdin.com/api/v2/#operation/api.projects.screenshots.post)
- [Edit Screenshot](https://developer.crowdin.com/api/v2/#operation/api.projects.screenshots.patch)
- [Get Screenshot](https://developer.crowdin.com/api/v2/#operation/api.projects.screenshots.get) (response)

### PR DESCRIPTION
**Pull request by Mahhheshh**
closes #139 

This PR Addes/Updates. 

The optional `labelIds` parameter has been added to `add_screenshot`. tests for adding screenshot has been updated to test the changes.

In `crowdin_api/api_resources/screenshots/types.py` the type hinting for `value` has been changed from `Any` to `str`. [docs](https://developer.crowdin.com/api/v2/#operation/api.projects.screenshots.patch)

### CODE REVIEW

**Comment by andrii-bodnar** [comment]
Hi @Mahhheshh, could you please address the CI issues? Thanks.

**Comment by andrii-bodnar** [comment]
@Mahhheshh could you please also update the [List Screenshots](https://developer.crowdin.com/api/v2/#operation/api.projects.screenshots.getMany) interface?

**Comment by Mahhheshh** [comment]
Sure I will update the List Screen Shots interface also.

**Comment by Mahhheshh** [comment]
It seems that I misinterpreted the documentation.

File: crowdin_api/api_resources/abstract/resources.py [hunk 1]
```diff
@@ -30,23 +30,39 @@ def get_page_params(
         page: Optional[int] = None,
         offset: Optional[int] = None,
         limit: Optional[int] = None,
+        labelIds: Optional[Iterable[int]] = None,
+        excludeLabelIds: Optional[Iterable[int]] = None
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/abstract/resources.py:34]: This is wrong, the base abstract resource shouldn't be affected. The labels are not page parameters

File: crowdin_api/api_resources/screenshots/resource.py [hunk 2]
```diff
@@ -34,6 +34,8 @@ def list_screenshots(
         page: Optional[int] = None,
         offset: Optional[int] = None,
         limit: Optional[int] = None,
+        labelIds: Optional[Iterable[int]] = None,
+        excludeLabelIds: Optional[Iterable[int]] = None
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/screenshots/resource.py:38]: please also add the `stringId` parameter here and put the new parameters after the `projectId`

File: crowdin_api/api_resources/screenshots/resource.py [hunk 3]
```diff
@@ -47,7 +49,8 @@ def list_screenshots(
         return self._get_entire_data(
             method="get",
             path=self.get_screenshots_path(projectId=projectId),
-            params=self.get_page_params(page=page, offset=offset, limit=limit),
+            params=self.get_page_params(
+                page=page, offset=offset, limit=limit, labelIds=labelIds, excludeLabelIds=excludeLabelIds),
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/screenshots/resource.py:53]: this is not a page params, it's query params of the request

example - https://github.com/crowdin/crowdin-api-client-python/blob/main/crowdin_api/api_resources/projects/resource.py#L64

File: crowdin_api/api_resources/screenshots/resource.py [hunk 4]
```diff
@@ -56,6 +59,7 @@ def add_screenshot(
         name: str,
         projectId: Optional[int] = None,
         autoTag: Optional[bool] = None,
+        labelIds: Optional[Iterable[int]] = None
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/screenshots/resource.py:62]: please also include the other missing parameters (fileId, branchId, directoryId)

File: crowdin_api/api_resources/screenshots/resource.py [hunk 5]
```diff
@@ -66,14 +70,19 @@ def add_screenshot(
 
         projectId = projectId or self.get_project_id()
 
+        request_data = {
+            "storageId": storageId,
+            "name": name,
+            "autoTag": autoTag,
+        }
+
+        if labelIds is not None:
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/screenshots/resource.py:79]: there is no need to check if the parameter exists. Check the similar method - https://github.com/crowdin/crowdin-api-client-python/blob/main/crowdin_api/api_resources/source_files/resource.py#L193

File: crowdin_api/api_resources/screenshots/resource.py [hunk 6]
```diff
@@ -88,7 +97,8 @@ def get_screenshot(self, screenshotId: int, projectId: Optional[int] = None):
 
         return self.requester.request(
             method="get",
-            path=self.get_screenshots_path(projectId=projectId, screenshotId=screenshotId),
+            path=self.get_screenshots_path(
+                projectId=projectId, screenshotId=screenshotId),
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/screenshots/resource.py:101]: please revert this change

File: crowdin_api/api_resources/screenshots/resource.py [hunk 7]
```diff
@@ -113,7 +123,8 @@ def update_screenshot(
                 "storageId": storageId,
                 "name": name,
             },
-            path=self.get_screenshots_path(projectId=projectId, screenshotId=screenshotId),
+            path=self.get_screenshots_path(
+                projectId=projectId, screenshotId=screenshotId),
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/screenshots/resource.py:127]: and this, and below

**Review by andrii-bodnar — CHANGES_REQUESTED** [review]

File: crowdin_api/api_resources/screenshots/resource.py (hunk 2 above)
Comment (andrii-bodnar) [reply crowdin_api/api_resources/screenshots/resource.py:39]: the labelIds and excludeLabelIds also put after the stringId

File: crowdin_api/api_resources/abstract/resources.py [hunk 8]
```diff
@@ -1,5 +1,5 @@
 from abc import ABCMeta
-from typing import Optional
+from typing import Optional, Iterable
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/abstract/resources.py:2]: please rollback all the changes in this file

**Review by andrii-bodnar — CHANGES_REQUESTED** [review]

**Review by andrii-bodnar — APPROVED** [review]
@Mahhheshh thank you!

**Pull request merged**